testsuite/features/step_definitions/navigation_steps.rb

```diff
@@ -532,6 +532,7 @@
 
   find(:xpath, "//header//i[@class='fa fa-sign-out']").click if all(:xpath, "//header//i[@class='fa fa-sign-out']", wait: 0).any?
 
+  raise 'Login page is not correctly loaded' unless has_field?('username')
```

Comment:
what if we wait until the 'username' field loads instead of the raise? :thinking:

Reply:
has_XXX? method are waiting. The timeout will be the CapybaraTimeout. In PRV BV, it will be 60 secs.